Source: https://src.fedoraproject.org/rpms/libgda5/blob/f43/f/libgda5-configure-c99-2.patch

The probe accidentally fails if the compiler enforces C type
rules because the first argument to mdb_bind_column is not of
the correct type.

Upstream no longer has this build system feature probe.

diff --git a/m4/mdbtools.m4 b/m4/mdbtools.m4
index 448c550f7d3e1119..7800c664b2779664 100644
--- a/m4/mdbtools.m4
+++ b/m4/mdbtools.m4
@@ -197,8 +197,8 @@ int main() {
 	    AC_COMPILE_IFELSE([AC_LANG_SOURCE([[
 #include <mdbtools.h>
 int main() {
-	MdbHandle *mdb;
-	int c;
+	MdbTableDef *mdb = 0;
+	int c = 0;
 	char *bound_data[256];
 	int len;
 	mdb_bind_column(mdb, c, bound_data[c], &len);

